[DO NOT MERGE]  Dev Get Start Content Experiment (#4292)
OfficeDev/office-js-docs-pr: docs/overview/office-add-ins-experiment.md

For an overview of Outlook add-ins, see [Outlook add-ins overview](../outlook/outlook-add-ins-overview.md).

### Create new objects in Office documents

You can embed web-based objects called content add-ins within Excel and PowerPoint documents. With content add-ins, you can integrate rich, web-based data visualizations, media (such as a YouTube video player or a picture gallery), and other external content.

![Embed web-based objects called content add-ins.](../images/about-addins-contentaddin.png)

## Office JavaScript APIs

The Office JavaScript APIs contain objects and members for building add-ins and interacting with Office content and web services. There's a common object model that's shared by Excel, Outlook, Word, PowerPoint, OneNote, and Project. There are also more extensive application-specific object models for Excel and Word. These APIs provide access to well-known objects such as paragraphs and workbooks, which makes it easier to create an add-in for a specific application.

## Code samples

> Discover practical add-in use cases and learn through hands-on scenario samples.

Reviewer: ```suggestion
Discover how Office Add-ins create solutions for content creation, better collaboration with peers, and task automation.
```

> Discover how Office Add-ins create solutions for content creation, better collaboration with peers, and task automation.

Reviewer: ```suggestion
Discover how Office Add-ins create solutions for content creation, better collaboration with peers, and task automation. Run the following samples to experience quick and easy real-life solutions in action.
```

- [[Word] AI Content Generation](https://github.com/OfficeDev/Word-Scenario-based-Add-in-Samples/tree/main/Word-Add-in-AIGC)
- [[Excel] Mail Merge](https://github.com/OfficeDev/Excel-Scenario-based-Add-in-Samples/tree/main/Mail-Merge-Sample-Add-in)
- [[PowerPoint] Shape Editing](https://learn.microsoft.com/en-us/office/dev/add-ins/powerpoint/shapes)
- [[Outlook] Set Signature](https://github.com/OfficeDev/Office-Add-in-samples/tree/main/Samples/outlook-set-signature)

> You can also start with the simplest hello-world add-ins with only a manifest, HTML web page, and a logo.

Reviewer: ```suggestion
When you're ready to create your own solution, start with a simple "Hello World" add-in with only a manifest, HTML web page, and a logo.
```
